pkgs/top-level/all-packages.nix

```
@@ -9524,6 +9524,8 @@ with pkgs;
 
   pngquant = callPackage ../tools/graphics/pngquant { };
 
+  poac = callPackage ../development/tools/poac { };
```

If by any reason Clang is mandatory instead of the default GCC, use this:
```
  poac = callPackage ../development/tools/poac {
     stdenv = llvmPackages.stdenv;
  };
```

or, more elegantly,

```suggestion
  poac = callPackage ../development/tools/poac {
    inherit (llvmPackages) stdenv;
  };
```

---

@AndersonTorres 

I am sorry to be late to respond.

I do not need to use Clang, but GCC `11` or later is required. When I omit to specify any compiler, Clang `11`, which cannot compile Poac, is used on `aarch64-darwin`. Could you please tell me how I use the default GCC?

```console
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/share/doc/poac -DCMAKE_INSTALL_INFODIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/include -DCMAKE_INSTALL_SBINDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/y1ijzmik13ak5vpb19rqsdkbv2lm2imb-clang-wrapper-11.1.0/bin/strip -DCMAKE_RANLIB=/nix/store/6651yv1y4iv29ag28sj7cs5bkm7jka0c-cctools-binutils-darwin-949.0.1/bin/ranlib -DCMAKE_AR=/nix/store/6651yv1y4iv29ag28sj7cs5bkm7jka0c-cctools-binutils-darwin-949.0.1/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/sn9y8vp3r4x09933pg490n9faplanbnx-poac-0.3.7 -DCPM_USE_LOCAL_PACKAGES=ON 
-- The CXX compiler identification is Clang 11.1.0
```

---

`stdenv = gcc<version required>Stdenv;` should work.

---

I see. Thank you :)